src/GCPClientManager.php

```diff
@@ -122,6 +135,7 @@ public function getBigQueryClient(string $runId, GenericBackendCredentials $cred
         return new BigQueryClientWrapper($runId, [
             'keyFile' => CredentialsHelper::getCredentialsArray($credentials),
             'restRetryFunction' => Retry::getRetryDecider($this->logger),
+            'requestTimeout' => self::TIMEOUT,
```

Comment:
vsade sa pise
```
type float $requestTimeout Seconds to wait before timing out the
     *           request. **Defaults to** `0` with REST and `60` with gRPC.
```
tak mi z toho vychadza ze je to fakt na request a nie na tu query, co myslis?